### ISSUE TITLE
can't reproduce the result

### ISSUE THREAD

**Issue by Ideal-Bushgorman** [ShuchangYe-bib/SGSeg#2]
Hi, thank you for sharing the code and your excellent work.

I tried to reproduce the experimental results reported in your paper, but I observe a ~2% lower Dice score compared to what was stated. I’ve followed the instructions in the README, including the data preprocessing and training settings. Could you kindly clarify if there are any additional steps, hyperparameter settings, or environment-specific factors that may influence the results?

<img width="615" height="678" alt="Image" src="https://github.com/user-attachments/assets/ecdda386-5c79-40c4-9bdb-a6a117f42230" />

**Comment by ShuchangYe-bib** [comment]
@Devil-Ideal, We appreciate your interest in our work.

A ~2% variation in Dice score can occur due to several factors:

Random initialization and training variance – Even with the same settings, stochasticity in weight initialization, data shuffling, and augmentation can introduce fluctuations in the results.

Environment differences – Versions of CUDA, PyTorch, and other dependencies may slightly affect reproducibility. We recommend checking that your environment matches the one specified in the paper/repo as closely as possible.

You are encouraged to report your reproduced result in your paper, as long as all experiments are run in the same environment.
